roles/simulator/tasks/start.yml

```diff
@@ -1,39 +1,96 @@
 ---
 
-- debug:
-    msg: "{{ ROOT_DIR }}"
-  tags:
-    - ueransim
-
 - name: copy config files to ueransim host
   template:
     src: "{{ ROOT_DIR }}/{{ item.1 }}"
     dest: ueransim_onramp/config/
   with_indexed_items: "{{ ueransim.servers[lookup('ansible.utils.index_of', groups['ueransim_nodes'], 'eq', inventory_hostname)] }}"
   when: inventory_hostname in groups['ueransim_nodes']
-  tags:
-    - ueransim
 
 - name: Run gNB
-  shell: nohup ./nr-gnb -c "../config/{{ item.1 }}" > /dev/null 2>&1 &
+  shell: nohup ./nr-gnb -c "../config/custom-gnb.yaml" > /dev/null 2>&1 &
```

Comment:
I do not think this change is correct because it will work only for a single `ueransim` instance. If the user wants to deploy multiple `ueransim` instances, the same file will be copied to the different instances, which is incorrect. I think this has to be kept as it was before.

Reply:
A different file is provided as a parameter in `servers` for each instance. The corresponding file for each instance will be copied, but all will be renamed with a standard file name.

Reply:
Am I overlooking at it, but I do not see where the files are being renamed. Can you please point me to where the files are renamed? Thanks!

Reply:
We are copying at line [11](https://github.com/opennetworkinglab/aether-ueransim/pull/16/files#diff-34f4c95e634d250e80fcc7ce42a325bf345cc1e1777774952fd8bfe0cd19be1fL11), but I see the issue, let me fix it.